docs/custom.css

```diff
@@ -44,8 +44,7 @@ body {
 
 /* Code Blocks & Syntax */
 .markdown-body .highlight pre, .markdown-body pre {
-  background-color: var(--color-bg-canvas-tertiary);
-  overflow: inherit
+  background-color: var(--color-bg-canvas-tertiary)
```

Comment:
I don't want the overflow to break on lines of code since that can give the reader the wrong impression of the code. We should rather scroll the container so that we fix the issue but keep the code examples correct.

https://user-images.githubusercontent.com/318208/117643128-fa1b7b00-b17f-11eb-90e2-6fc33e039489.mov

```suggestion
  background-color: var(--color-bg-canvas-tertiary)
  overflow: scroll;
```

Reply:
@koddsson 
Sorry for the late reply.

I was going to define it as overflow at first.
However, since `overflow: auto;` in the following code adapted from https://github.github.io/src/markdown/code.scss is applied, I thought it was unnecessary.

```css
.markdown-body .highlight pre, .markdown-body pre {
    padding: 16px;
    overflow: scroll;
    font-size: 85%;
    line-height: 1.45;
    background-color: #f6f8fa;
    border-radius: 3px;
}
```

If the content overflows, a scroll bar will appear.

Reply:
If you use only `overflow: scroll;`, the scroll bar will appear on the y-axis as well, which is not so good.

<img width="998" alt="Screen Shot 2021-05-13 at 21 45 01" src="https://user-images.githubusercontent.com/1996642/118127743-0845f180-b435-11eb-9d12-8a248a6f9733.png">

Reply:
> If you use only `overflow: scroll;`, the scroll bar will appear on the y-axis as well, which is not so good.

I don't get those scrollbars. Maybe you have `Show scroll bars` set to always showing them?

Here's my macOS settings for scrollbars:

![image](https://user-images.githubusercontent.com/318208/118285280-337a2080-b4c9-11eb-9e65-464fced26b0d.png)

Reply:
Yes, my settings are set to `Always`.

<img width="426" alt="Screen Shot 2021-05-15 at 11 03 45" src="https://user-images.githubusercontent.com/1996642/118345396-2adc2580-b56f-11eb-9241-1c19ce59a3c4.png">

However, this is not set incorrectly, but **because I want it to be always visible.**
This is my own choice.

Reply:
By the way, the scrollbar appears like this when viewed in Chronium Edge on windows 10.
In Windows OS, scrollbars cannot be hidden like in MacOS. (As far as I know).

![image](https://user-images.githubusercontent.com/1996642/118345494-c8375980-b56f-11eb-8270-62fecbb89e44.png)

To show scrollbars only where they are visible regardless of OS, `overflow: auto` , or `overflow-x: sroll` is suitable.

Reply:
> To show scrollbars only where they are visible regardless of OS, `overflow: auto` , or `overflow-x: sroll` is suitable.

Sure let's do `overflow: auto`.

Reply:
```suggestion
  background-color: var(--color-bg-canvas-tertiary);
  overflow: auto;
```